core/lib/zksync_core/src/block_reverter/mod.rs

```diff
@@ -333,6 +336,7 @@ impl BlockReverter {
         priority_fee_per_gas: U256,
         nonce: u64,
     ) {
+        const DEFAULT_ZKSYNC_CHAIN_ID: u128 = 270; // Default value if the environment variable is not set or cannot be parsed
```

Comment:
too risky. If env variable is not set  - and AFAIK this is a separate tool, right? - then I'd prefer to crash (assert) mentioning that this is a wrong config.